recipe/meta.yaml

```
@@ -12,34 +14,72 @@ source:
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv "
-  skip: True  # [osx or (not py36) or (win and vc<14)]
-
-requirements:
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-  host:
-    - python
-    - pytorch
-    - setuptools
-    - pip
-  run:
-    - python
-    - pytorch
-    - cxxfilt  # [linux]
-    - tqdm
-    - numpy
-    - PyYAML
-    - pytest
-
-test:
-  imports:
-    - apex
-    - apex.amp
-    - apex.parallel
-    - apex.optimizers
-    - apex.normalization.fused_layer_norm
+  skip: True  # [not linux or py2k]
+
+outputs:
+  - name: nvidia-apex-proc
+    version: {{ version }}
+    build:
+      number: 0
+      string: "{{ nvidia_apex_proc_type }}"
+    test:
+      commands:
+        - exit 0
+    about:
+      home: https://github.com/conda-forge/nvidia-apex-feedstock
+      license: BSD-3-Clause
+      license_family: BSD
+      summary: A meta-package to select CPU or GPU nvidia-apex build.
+
+  - name: nvidia-apex
+    version: {{ version }}
+    build:
+      number: 0
+      script: python -m pip install . -vv
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
```

Would constrain this to Linux as that is the only platform supported by the CUDA compiler currently. Noting this especially here as we will need to skip this on Windows.

```suggestion
        - {{ compiler('cuda') }}  # [linux64 and cuda_compiler_version != "None"]
```

---

We probably also need this to fix the Windows build error.